### DISCUSSION TITLE
Rejoin at player's logged off position

### DISCUSSION THREAD

**Discussion by Squidtito** [Pryaxis/TShock#2644] [Q&A] [answered]
I was wondering how to spawn players back where they logged off, I've been really confused on how to fix this.

**Comment by FrostBird347** [comment, marked as answer]
This is somewhat possible through the [server's config](https://tshock.readme.io/docs/config-settings) by setting `RememberLeavePos` to `true`.

Unfortunately this does not persist past a server restart and I was unable to find a plugin which added this functionality in.
In addition, #2871 seems to have completely broken this option.

**Reply by Squidtito** [reply]
I hope this issue can be fixed. And added functionality to save the player's position even after a server restart.
